R/webshot.R

```diff
@@ -53,6 +53,7 @@ NULL
 #'   device (but using zoom will not report that there is a HiDPI device).
 #' @param useragent The User-Agent header used to request the URL.
 #' @param max_concurrent (Currently not implemented)
+#' @param verbose Show / Hide console messages
```

Comment:
In these situations, where the function emits status messages by default, I'd prefer to name the argument `quiet` so that users opt-in via `quiet = TRUE`.

```suggestion
#' @param quiet If `TRUE`, status updates via console messages are suppressed.
```

I typically reserve `verbose` for situations where many detailed, debug-level messages are printed. If I were newly approaching `webshot()`, I'd expect `verbose = TRUE` to cause chromote to also print debug messages.